fluster/utils.py

```diff
@@ -166,9 +166,15 @@ def _win_user_data_dir(appname: str) -> str:
     return os.path.join(path, appname)
 
 
+def _win_site_data_dirs(appname: str) -> List[str]:
+    """Return data directory shared by users"""
+    # On Windows always user_data_dir
+    return [_win_user_data_dir(appname)]
```

Comment:
I might be missing something but doesn't this return a List of 1 string always?

Reply:
No implementation of `_site_data_dirs` for Windows.

https://github.com/fluendo/fluster/blob/2e8d5abd61160d9fcc1469060105b7571669f84f/fluster/utils.py#L169-L174